src/javascript/app_2/pages/settings/settings_data.js

```diff
@@ -0,0 +1,79 @@
+import AccountPassword        from './sections/account_password.jsx';
+import ApiToken               from './sections/api_token.jsx';
+import AuthorizedApplications from './sections/authorized_applications.jsx';
+import CashierPassword        from './sections/cashier_password.jsx';
+import FinancialAssessment    from './sections/financial_assessment.jsx';
+import Limits                 from './sections/limits.jsx';
+import LoginHistory           from './sections/login_history.jsx';
+import PersonalDetails        from './sections/personal_details.jsx';
+import SelfExclusion          from './sections/self_exclusion.jsx';
+import { localize }           from '../../../_common/localize';
+
+const data = [
+    {
+        title: localize('Profile'),
+        items: [
+            {
+                title      : localize('Personal Details'),
+                description: localize('View your personal information.'),
+                img_src    : '../images/trading_app/settings/ic-personal-details.svg',
```

Comment:
`Url.urlForStatic()` returns a URL based on the project root, passing a relative address may cause unresolved paths. Just `'images/...'` should be fine.